tests/test_upload.py

```diff
@@ -79,12 +79,12 @@ def test_get_config_old_format(tmpdir):
         """))
 
     try:
-        upload.upload(dists=dists, repository="pypi", sign=None, identity=None,
-                      username=None, password=None, comment=None,
-                      cert=None, client_cert=None,
-                      sign_with=None, config_file=pypirc, skip_existing=False,
-                      repository_url=None, verbose=None,
-                      )
+        upload_settings = settings.Settings(
```

Comment:
This test shows that we've essentially made our validation of configuration settings/values more common than just the upload command. Future commits should rewrite this test and move it to `tests/test_settings.py`.